react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -95,99 +100,55 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
           validationSchema={validationSchema}
           onSubmit={handleSubmit}
         >
-          {/*TODO_REACT: Will change to core-wrapper's FieldWrapperFormik instead. https://tacc-main.atlassian.net/browse/WG-246 */}
-          {({ errors, touched, values, handleChange }) => (
+          {() => (
             <Form>
```

Comment:
**I am sorry.** Some global `c-form…` classes will expect this "parent block"[^1] class to exist.

```suggestion
            <Form class="c-form">
```

Should there be a `<FormFormik>` for this? It would be consistent.

[^1]: The naming structure is a CSS community methodology called BEM (Block, Element, Modifier). Documentation: [internal](https://tacc-main.atlassian.net/wiki/x/xghv), [external](https://en.bem.info/methodology/quick-start/). [↩](https://github.com/TACC-Cloud/hazmapper/pull/208/files#user-content-fnref-1-f9ca5be8fcb41acfa982fb44028de2fa)